docs/apps/lnw/es2k/es2k-lnw-overlay-vms.md

```diff
@@ -253,8 +253,8 @@ Example:
 
   ```bash
      # Create a source port for an overlay VF (VSI-27). Source port value should be VSI ID + 16.
-      p4rt-ctl add-entry br0 linux_networking_control.tx_source_port_v4 \
-      "vmeta.common.vsi=27/2047,action=linux_networking_control.set_source_port(43)"
+      p4rt-ctl add-entry br0 linux_networking_control.tx_source_port_v4
```

Comment:
It is important that command text be literally correct. This is intentionally written (for improved readability) as a continued command. There must be a backslash at the end of line 256, since these would be two separate (invalid) command lines without it.

The other edits are correct. They don't delete the backslash.

Reply:
Fixed